src/main/java/redis/clients/jedis/commands/JedisCommands.java

```diff
@@ -650,6 +651,18 @@ default String setex(String key, int seconds, String value) {
   List<StreamEntryID> xclaimJustId(String key, String group, String consumername, long minIdleTime,
       XClaimParams params, StreamEntryID... ids);
 
+  /**
+   * XAUTOCLAIM key group consumer min-idle-time start [COUNT count] [JUSTID]
+   */
+  Map.Entry<StreamEntryID, List<StreamEntry>> xautoclaim(String key, String group, String consumerName,
+      long minIdleTime, StreamEntryID start, XAutoClaimParams params);
+
+  /**
+   * XAUTOCLAIM key group consumer min-idle-time start [COUNT count] [JUSTID]
+   */
+  Map.Entry<StreamEntryID, List<StreamEntryID>> xautoclaimJustId(String key, String group, String consumerName,
+      long minIdleTime, StreamEntryID start, XAutoClaimParams params);
```

Comment:
In JavaDoc, please specify that `start` param can be `null` without any issue. (This is handled in Client class.)

PS: This is not true for binary class.